includes/admin/class-wc-admin-notices.php

```diff
@@ -212,17 +207,6 @@ public static function install_notice() {
 		include( 'views/html-notice-install.php' );
 	}
 
-	/**
-	 * Show the Theme Check notice.
-	 */
-	public static function theme_check_notice() {
```

Comment:
Does this need to be deprecated? I can't imagine anyone is using it for anything, but it is public.

Reply:
It's internal, but still public method.
Better keep and throw a warning for now.